code/game/objects/items/ego_weapons/non_abnormality/rcorp.dm

```diff
@@ -34,3 +34,223 @@
 	to_chat(user, "<span class='notice'>\The [src] will now deal [damtype] damage.</span>")
 	playsound(src, 'sound/items/screwdriver2.ogg', 50, TRUE)
 
+//Command Sabre
+/obj/item/ego_weapon/city/rabbit_blade/command
+	name = "R-corp command sabre"
+	desc = "A stronger rcorp blade made for the ground commander."
+	icon_state = "rcorp_sabre"
+	inhand_icon_state = "multiverse"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	force = 70
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 100,
+							TEMPERANCE_ATTRIBUTE = 0,
+							JUSTICE_ATTRIBUTE = 100
+							)
+
+/obj/item/ego_weapon/city/rabbit_blade/command/attack_self(mob/living/user)
+	switch(damtype)
+		if(RED_DAMAGE)
+			damtype = WHITE_DAMAGE
+		if(WHITE_DAMAGE)
+			damtype = BLACK_DAMAGE
+		if(BLACK_DAMAGE)
+			damtype = PALE_DAMAGE
+		if(PALE_DAMAGE)
+			damtype = RED_DAMAGE
+	armortype = damtype // TODO: In future, armortype should be gone entirely
+	to_chat(user, "<span class='notice'>\The [src] will now deal [damtype] damage.</span>")
+	playsound(src, 'sound/items/screwdriver2.ogg', 50, TRUE)
```

Comment:
This should be inherited entirely from it's parent type (rabbit_blade) and be unnecessary